Jenkinsfile-rf-pipeline

```diff
@@ -0,0 +1,312 @@
+env.ghprbGhRepository = env.ciPipelineUtilsRepo ?: 'CentOS-PaaS-SIG/ci-pipeline'
+env.ghprbActualCommit = env.ciPipelineUtilsCommit ?: 'master'
```

Comment:
Don't you want this to be:

env.ghprbGhRepository = env.ghprbGhRepository ?: 'CentOS-PaaS-SIG/ci-pipeline'
env.ghprbActualCommit = env.ghprbActualCommit ?: 'master'

Reply:
yeah, typo on my part.